makedeps/package.json

```diff
@@ -7,11 +7,16 @@
     "make": "./node_modules/.bin/browserify index.js > ../assets/bundle.js"
   },
   "dependencies": {
+    "@babel/plugin-transform-react-jsx": "^7.13.12",
```

Comment:
I'm confused why this is here -- there's no jsx in these deps? all the jsx lives in react-ui?

Reply:
Sorry might, be a leftover from my babysteps into this project.
I will remove..